dali/operators/decoder/image_decoder.cc

```diff
@@ -24,79 +24,114 @@ DALI_SCHEMA(ImageDecoderAttr)
-in runtime. Ignored when `split_stages` is false.)code",
+      R"code(**Experimental**, applies **only** to the ``mixed`` backend type.
+
+Uses the chunk pinned memory allocator and allocates a chunk of the
+``batch_size*prefetch_queue_depth`` size value during the construction and suballocate
+them in runtime. When ``split_stages`` is false, this argument is ignored.)code",
       false)
   .AddOptionalArg("use_fast_idct",
-      R"code(Enables fast IDCT in CPU based decompressor when GPU implementation cannot handle given image.
-According to libjpeg-turbo documentation, decompression performance is improved by 4-14% with very little
-loss in quality.)code",
+      R"code(Enables fast IDCT in a CPU-based decompressor when the GPU implementation cannot
+handle an  image.
+
+According to the libjpeg-turbo documentation, decompression performance is improved by up to 14%
+with a low reduction in quality.)code",
       false)
   .AddOptionalArg("memory_stats",
-      R"code(**`mixed` backend only** Print debug information about nvJPEG allocations.
-The information about the largest allocation might be useful to determine suitable values for
-`device_memory_padding` and `host_memory_padding` for a given dataset.
+      R"code(Applies **only** to the ``mixed`` backend type.
+
+Prints the debug information about nvJPEG allocations. The information about the largest
+allocation might be useful to determine suitable values for ``device_memory_padding`` and
+``host_memory_padding`` for a dataset.
 
-Note: The statistics are global for the whole process (and not per operator instance) and include
-the allocations made during construction (when the padding hints are non-zero).)code",
+.. note::
+  The statistics are global for the entire process, not per operator instance, and includes
+  the allocations made during construction when the padding hints are non-zero.
+)code",
       false);
 
 DALI_SCHEMA(ImageDecoder)
-  .DocStr(R"code(Decode images
+  .DocStr(R"code(Decodes the images.
 
-For jpeg images, the implementation will use *nvJPEG* library or *libjpeg-turbo* depending on the
-selected backend (*mixed* and *cpu* respectively). Other image formats are decoded with *OpenCV* or
-other specific libraries (e.g. *libtiff*).
+For jpeg images, depending on the backend you select (mixed and cpu), the implementation uses
+the *nvJPEG* library or *libjpeg-turbo*, respectively . Other image formats are decoded
+with *OpenCV* or other specific libraries, such as *libtiff*).
 
-If used with *mixed* device, the operator will use a dedicated hardware decoder if available.
+If used with a mixed device, and the hardware is available, the operator will use
+a dedicated hardware decoder.
 
-The output of the decoder is in *HWC* layout.
+The output of the decoder is in the *HWC* layout.
 
-Supported formats: JPG, BMP, PNG, TIFF, PNM, PPM, PGM, PBM.)code")
+The following formats are supported:
+
+* JPG
+* BMP
+* PNG
+* TIFF
+* PNM
+* PPM
+* PGM
+* PBM)code")
```

Comment:
In my opinion, this list is not important enough to justify it occupying so much vertical space. It shifts attention from important information, which is likely to be needed repeatedly (for beginners, with each use of the operator) to something that's likely to be necessary once or never, since most common formats _are_ supported.

Reply:
Done